src/mocks/users/index.js

```diff
@@ -80,4 +81,34 @@ export default [
       }),
     );
   }),
+
+  rest.get(`${API_HOST}/users`, (req, res, ctx) => {
+    const isVerified = req.url.searchParams.get('isVerified') === 'true';
+
+    if (!isVerified && typeof isVerified !== 'boolean') {
```

Comment:
Probably, a condition `typeof isVerified !== 'boolean'` is redundant because it's always boolean.